auth-web/src/store/modules/business.ts

```diff
@@ -163,23 +163,28 @@ export default class BusinessModule extends VuexModule {
           return null
         })
 
-      if (resp?.data?.affiliationInvites) {
-        const affiliationInviteInfos: AffiliationInviteInfo[] = resp.data.affiliationInvites
+      if (resp?.data?.affiliationInvitations) {
```

Comment:
```
const pendingAffiliationInvites = resp?.data?.affiliationInvitations || [];

for (const affiliationInviteInfo of pendingAffiliationInvites) {
  const isFromOrg = AffiliationInviteInfo.isFromOrg(affiliationInviteInfo, this.currentOrganization.id);
  const isToOrgAndActive = AffiliationInviteInfo.isToOrgAndActive(affiliationInviteInfo, this.currentOrganization.id);
  const isAccepted = AffiliationInviteInfo.isAccepted(affiliationInviteInfo);

  if (isFromOrg && !isAccepted) {
    const newBusiness = { ...affiliationInviteInfo.entity, affiliationInvites: [affiliationInviteInfo] };
    affiliatedEntities.push(newBusiness);
  } else {
    const business = affiliatedEntities.find(business => business.businessIdentifier === affiliationInviteInfo.entity.businessIdentifier);

    if (business) {
      if (isToOrgAndActive || isFromOrg) {
        business.affiliationInvites = (business.affiliationInvites || []).concat(affiliationInviteInfo);
      }
    }
  }
}
```

Reply:
didnt use this exact snipet, but did some updates along those lines.